src/cata_tiles.h

```diff
@@ -3,6 +3,7 @@
 
 #include <SDL.h>
 #include <SDL_ttf.h>
+#include <SDL/SDL_gpu.h>
```

Comment:
Could this be change just to `#include <SDL/SDL_gpu.h>` like it is done for `#include <SDL.h>`?
This breaks building with frameworks.